runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -28,7 +28,9 @@
 #include "objhelp.h"
 #include "rommeth.h"
 #include "vmaccess.h"
+#if JAVA_SPEC_VERSION < 24
 #include "java_lang_Class.h"
+#endif /* JAVA_SPEC_VERSION < 24 */
```

Comment:
I don't think this include should be conditional. Ideally, it would be first (unconditionally), followed by includes needed for the implementation.

Reply:
I removed this exclude and fixed the test failure.